packages/compass-sidebar/.mocharc.js

```diff
@@ -0,0 +1,8 @@
+const base = require('@mongodb-js/mocha-config-compass/compass-plugin');
+
+module.exports = {
+  ...base,
+  require: base.require.concat([
+    'test/setup.js'
```

Comment:
I'll keep on eye on this soon if I fix up the tests. Might not be necessary going forward.